chaincfg/params.go

```diff
@@ -288,6 +289,8 @@ var MainNetParams = Params{
 
 	CosmicInflationActivationTime: 1652616000,
 
+	Upgrade9ForkHeight: 792772, // 000000000000000002B678C471841C3E404EC7AE9CA9C32026FE27EB6E3A1ED1
```

Comment:
Is there a reason to define this only on mainnet?

Reply:
No, it's just something to be added. I'll add it.